### PR TITLE
(closes #1025) remove travis

### LINKED ISSUE #1025: Turn-off Travis
travis-ci.org is due to be switched off at the end of this month and it already takes ~1hr to run the tests c.f. 15mins on github actions. We will therefore stop running tests on Travis (by just deleting the yml file).

### PR DESCRIPTION
**Pull request by arporter**

### CODE REVIEW

**Comment by arporter** [comment]
This is ready for review. I think the continuous-integration/travis-ci check above is still present because I hadn't disabled the Travis webhook. I've done that now.

**Comment by arporter** [comment]
Just a note to say I've updated the wiki pages (reviewing and releasing instructions) to remove mentions of Travis.

**Comment by arporter** [comment]
I realised I needed to edit the settings relating to the protection of the 'master' branch on GitHub. I've now made the various Actions 'required' and removed that tag from the Travis-related checks.

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -67,7 +67,8 @@ jobs:
         # Some of the examples use Jupyter
         pip install jupyter
         # Install the version of fparser pointed to by the submodule
-        pip install external/fparser
+        # instead of the released version (from pypi).
+        # pip install external/fparser
```
Comment (rupertford) [inline .github/workflows/python-package.yml:71]: Change the comment to "If you want to install the version of .... (from pypi) then uncomment the fparser install line below".

File: doc/developer_guide/working_practises.rst [hunk 2]
```diff
@@ -302,45 +302,44 @@ These files must be located in the same directory as the kernels.
...
+and therefore the line described above must be commented out again
 before making a release.
 
-Given that a run of the test-suite on Travis uses approximately 45
-minutes of CPU time, it is good practise to avoid triggering it
-unnecessarily (e.g. if you know that a certain commit won't
-pass). This can be achieved by appending "[skip ci]" (without the
-quotes) to the end of the associated git commit message.
+Given that a run of the test-suite on GitHub Actions uses
+approximately 45 minutes of CPU time, ideally we would avoid
```
Comment (rupertford) [inline doc/developer_guide/working_practises.rst:339]: Its 15 minutes isn't it?

**Review by rupertford — CHANGES_REQUESTED** [review]
Looks good. I just have one small inline request and could you also please take a look at the reference to Travis in  `examples/common.mk` and remove.

**Comment by arporter** [comment]
I thought I'd found every mention of Travis! I've fixed the one in common.mk and addressed your other comments.

**Review by rupertford — APPROVED** [review]
All issues have been addressed. CI and codecov are happy. Approving.

**Pull request merged**